### ISSUE TITLE
Accept ToS

### ISSUE THREAD

**Issue by brettgus** [xssc/gimme-ptc#12]
Have you made changes to accomodate this?
https://www.reddit.com/r/pokemongodev/comments/4wk3e4/psa_new_pogo_accounts_have_to_accept_tos_ingame/

**Comment by xssc** [comment]
It's being worked on

**Comment by maaim** [comment]
The tos.py script here works very well.  https://github.com/TBTerra/spawnScan

**Comment by Kyriten** [comment]
Also, [this](https://github.com/TachyonRSA/PoGoTos) works well too.

**Comment by brettgus** [comment]
Yeah, it's all kind of moot unless you're doing captcha solving though as it's required to create new accounts
